axon.go

```diff
@@ -87,20 +77,23 @@ func (a *Axon) Run() error {
 		return fmt.Errorf("unable to check target database version: %w", err)
 	}
 
-	// TODO: (1) add support for selecting the warp-pipe mode
-	// TODO: (2) only print the source stats if that is audit
-	sourceCount, err := a.printStats(sourceDBConn, "source")
-	if err != nil {
-		return fmt.Errorf("unable to get source db stats: %w", err)
-	}
-	targetCount, err := a.printStats(targetDBConn, "target")
-	if err != nil {
-		return fmt.Errorf("unable to get target db stats: %w", err)
-	}
-
-	if sourceCount == targetCount {
-		a.Logger.Info("changeset counts match")
-		return nil
+	// TODO: only print the source stats if that is audit
+	switch a.Config.ListenerMode {
+	case ListenerModeNotify:
+		sourceCount, err := a.printStats(sourceDBConn, "source")
+		if err != nil {
+			return fmt.Errorf("unable to get source db stats: %w", err)
+		}
+		targetCount, err := a.printStats(targetDBConn, "target")
+		if err != nil {
+			return fmt.Errorf("unable to get target db stats: %w", err)
+		}
+		if sourceCount == targetCount {
+			a.Logger.Info("changeset counts match")
+			return nil
+		}
+	case ListenerModeLogicalReplication:
+		// Do nothing
```

Comment:
i'd either add a default case that errors out or delete this case

Reply:
removed in 9a032168f4b9952b85e58d9ea6a0eb58563494e1 (not expecting anything here for LR mode)